src/hotspot/os_cpu/linux_aarch64/os_linux_aarch64.cpp

```diff
@@ -382,7 +382,14 @@ int os::extra_bang_size_in_bytes() {
 
 extern "C" {
   int SpinPause() {
-    return 0;
+    if (VM_Version::spin_wait_desc().inst() == SpinWait::NONE) {
+      return 0;
+    }
```

Comment:
Would it be safe and more efficient to test func for NULL?

Reply:
Good idea.

Reply:
Done